### PR TITLE
Migrate DevHub content from Old DevHub

### PR DESCRIPTION
**Pull request by jenreiher-bcgov**
This represents a very, very rough draft of all the things "in DevHub" and one version of how we might bring them into the BCDG in future. They are intentionally not linked in the IA yet. 

* Each of these pages represents a backlog item that will require collaboration with multiple other teams to complete. 
* We may not need to migrate all of these before retiring DevHub, but certainly our users will benefit from prioritizing some of these!

### CODE REVIEW

File: docs/choosing-technology/how-to-select-technology.md [hunk 1]
```diff
@@ -1,98 +1,28 @@
...
+
+## Determining suitability
```
Comment (jenreiher-bcgov) [inline docs/choosing-technology/how-to-select-technology.md:21]: my instinct says this should be above the chart - but defer to @Shivanganii

**Review by sheaphillips — APPROVED** [review]
I've completed my edits and pushed them, resolved conflicts with `main`, etc. This is now ready to merge as far as I'm concerned. We can look at any tweaks separately.

File: docs-drafts/ci-cd.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Commonly Used CI/CD Tools
+
+|  Tool |  Purpose | Support  | 
+|---|---|---|
+|  *Argo CD* |  GitOps continuous delivery tool for Kubernetes |  Availabe within OpenShift |
```
Comment (Shivanganii) [inline docs-drafts/ci-cd.md:5]: For the tools section specifically, does it need italics?

File: docs-drafts/ci-cd.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+* Improved stability and consistency
+
+* An improved security model
+
+* It is efficient, well supported, and well documented.
+
+* It can reduce the maintenance overhead of pipelines
+
+* It will help you to reduce resource consumption by using a shared service
```
Comment (Shivanganii) [inline docs-drafts/ci-cd.md:27]: Can we say, "It helps you to reduce..." > changing to active voice without changing meaning? The previous line meaning would change if it was active voice, not so sure about this one.

File: docs-drafts/ci-cd.md (hunk 2 above)
Comment (sheaphillips) [reply docs-drafts/ci-cd.md:5]: I wanted to provide increase the visibility of the items in the Tool column, but it's not strictly necessary, but maybe we could use **bold** instead if that's preferable?

File: docs-drafts/ci-cd.md (hunk 3 above)
Comment (sheaphillips) [reply docs-drafts/ci-cd.md:27]: For sure, no concerns with that change.

**Comment by sheaphillips** [comment]
@Shivanganii I just realized that there's an important bit of info about this pull request that isn't at all obvious - we are actually only concerned about a single, new file (choosing-technology.md). The other new/edited files are heavily under construction and several won't ever be released. Apologies for this - I should have provided that information sooner.

**Pull request merged**